### PR TITLE
Prototype side by side commands for getting started

### PR DESCRIPTION
**Pull request by digitarald**
Scaffolding out a basic implementation for opening new tabs from Getting Started in split view. Works best for commands that directly open a tab (like Settings), but I also added it to quick open just to try it out.

recording (3)

One todo is to not reset the split layout if a user is already in split mode, but there might be more ideas to avoid surprises.

To further learn, we could put this behind an experiment flag to see if this helps users complete more of getting started, but also use it for extension walkthroughs that have more content to open.

cc @misolori

### CODE REVIEW

File: src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts [hunk 1]
```diff
@@ -264,7 +264,14 @@ export class GettingStartedPage extends EditorPane {
 							}
 							const taskToRun = assertIsDefined(this.currentCategory?.content.items.find(task => task.id === argument));
 							if (taskToRun.button.command) {
-								this.commandService.executeCommand(taskToRun.button.command);
+								if (taskToRun.button.sideBySide) {
+									// todo: Only split when current layout is 1-column
+									this.commandService.executeCommand('workbench.action.editorLayoutTwoColumns')
+										.then(() => this.commandService.executeCommand('workbench.action.focusNextGroup'))
+										.then(() => this.commandService.executeCommand(taskToRun.button.command));
```
Comment (digitarald) [inline src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts:271]: Another todo: There is a TS error here that `command` might be `undefined`, which is odd as it is within the `if` and the same line works 2 lines below.

Comment (JacksonKearl) [reply src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts:271]: TS discards the inferred constraints once you're in a new function scope (the `.then` block could happen a long time after the `if`, in the mean time a different task could change the definedness). Not an actual issue here, but TS doesn't know that.

**Comment by misolori** [comment]
Love this!

**Comment by digitarald** [comment]
@JacksonKearl I assigned you initially as I just wanted to discuss the approach and wasn't sure if you had other ideas already. If you think this is really close as is, I can get it over the finish line.

**Comment by JacksonKearl** [comment]
Seems reasonable to me, I'll tweak it to fix the compile and similar but should be good. Will likely be all overhauled in move to md-powered descriptions, but good intermediary until then.

**Comment by JacksonKearl** [comment]
Neat idea, thanks!

**Pull request merged**